sfa_dash/templates/forms/cdf_forecast_download_form.html

```diff
@@ -1,43 +1,39 @@
-{% extends "dash/data.html" %}
-{# {% set sidebar = True %} #}
+{% extends "forms/base/download_form.html" %}
 {% set page_title = 'Download Forecast data' %}
-{% block content %}
-{{ metadata | safe }}
-<h3>Download data for Forecast 'DA Power'</h3>
-{#<p><b>Available Date Range:</b> <i>2016-01-01T12:00Z</i> to <i>2019-01-01T12:00Z</i></p>#}
-<form action="{{ url_for('forms.download_cdf_forecast_data', uuid=uuid) }}" method="post" id="forecast-download">
-  <div class="form-group">
-	<div class="form-element">
-	  <label for="start">Start time:</label>
-	  <input type="datetime-local" id="start" name="period-start">
-    </div>
-    <div class="form-element">
-	  <label for="end">End time:</label>
-	  <input type="datetime-local" id="end" name="period-end">
-    </div>
-	<div class="form-element full-width">
-	  <label for="format">Format</label><br/>
-	  <input type="radio" data-toggle="collapse" data-target=".download-example" name="format" value="text/csv" checked>CSV</input>
-	  <input type="radio" data-toggle="collapse" data-target=".download-example" name="format" value="application/json">JSON</input>
-	</div>
-	<div class="form-element full-width">
-	  <div class="download-example csv show">
-	  <p>CSV downloads will have the following format.</p>
-	  <pre class="example-text">
+{% set form_title = "Download data for Forecast 'DA Power'" %}
```

Comment:
This should be dynamic based on the metadata, no?

Reply:
is it supposed to be set when templating? Looks like that isn't working at least when I run locally. Maybe this shouldn't be set at all in the form since you have the conditional block anyway

Reply:
You're right, but I don't know that it's necessary in the first place. I neglected to set this dynamically after moving away from the hard-coded values. If we need this subtitle to exist reminding a user that they're downloading the data for the site with it's metadata at the top of the page, all we need to do is print the name dynamically. Without it the page title is 'Download <data-type>' and the metadata is shown directly below that so it is redundant.